reverse.py

```diff
@@ -0,0 +1,40 @@
+from string import punctuation, digits
+
```

Comment:
From task 1 I will start asking you to format your code to certain styling and code practices to shape your skills in a way that it will be appreciative by your potential team members